conman/redirects/tests/test_models.py

```diff
@@ -23,7 +24,8 @@ def test_fields(self):
 
     def test_target_self(self):
         """A RouteRedirect's target cannot be itself."""
-        redirect = ChildRouteRedirectFactory.create()
+        with ignore_django20_deprecation():
+            redirect = ChildRouteRedirectFactory.create()
```

Comment:
I'm very concerned about silencing warnings that we're not truly addressing, as it may hide issues.

If these warnings are raised from django-polymorphic, I'd rather see the warning that simply silence it.